sdk/machinelearning/azure-mgmt-machinelearningcompute/README.md

```diff
@@ -15,12 +15,6 @@ library.
 For a more complete set of Azure libraries, see the
 [azure](https://pypi.python.org/pypi/azure) bundle package.
```

Comment:
https://pypi.python.org/pypi/azure package is deprecated - this link should be replaced with https://aka.ms/azsdk/python/all

Reply:
isn't this README generated, though? should this be an update in autorest?

Reply:
you are right - we need to update in autorest too - but replacing few links now is less costlier than regenrating all the packages - that said, we need to fix in autorest

Reply:
can you please let me know what file in autorest needs to be updated?

Reply:
Is it this - https://github.com/Azure/azure-sdk-for-python/blob/8447aadb71cdaad69524785efd0dff18960d3076/tools/azure-sdk-tools/packaging_tools/templates/README.md#L3